Source/Core/Core/PowerPC/Jit64Common/EmuCodeBlock.cpp

```diff
@@ -215,14 +216,14 @@ class MMIOReadCodeGenerator : public MMIO::ReadHandlingMethodVisitor<T>
   {
   }
 
-  void VisitConstant(T value) override { LoadConstantToReg(8 * sizeof(T), value); }
+  void VisitConstant(T value) override { LoadConstantToReg(CHAR_BIT * sizeof(T), value); }
```

Comment:
Any particular reason why you didn't use `BitSize<T>()` instead? It seems to fit the same pattern as the others, at least on ~~paper~~ GitHub.

Reply:
I did at first, but this group of functions accept the bit size parameter as an int for reasons beyond my control / outside the scope of this PR.  This caused made MSVC throw a warning turned error about potential loss of information in converting from size_t to int.  This was the next best solution, then.